### PR TITLE
Remove Unicode 1 names and add Unicode Name Aliases

### PR DESCRIPTION
**Pull request by samcv**
This adds support in the codepointfromname op for all of the official
Unicode name aliases. It removes the old Unicode 1 names in the name
registry and replaces them with <control-xxxx> with the codepoint
number.

There are only 5 codepoints which have changed their functionality
aside from the 400+ Name Aliases added:

Correction:
  * BELL
    * Used to resolve to U+0007 which caused the 🔔 U+1F514 to be
      inaccessible.
    * The following can be used to access U+0007:
      ALERT, or BEL

Unicode 1 Name removed:
  * LINE FEED (LF):
    * The following can be used instead:
      LINE FEED, NEW LINE, END OF LINE, LF, NL or EOL
  * FORM FEED (FF)
    * The following can now be used instead:
      FORM FEED or FF
  * CARRIAGE RETURN (CR)
    * The following can now be used instead:
      CARRIAGE RETURN or CR
  * NEXT LINE (NEL)
    * The following can now be used instead:
      NEXT LINE or NEL

### CODE REVIEW

**Comment by MasterDuke17** [comment]
Typo, 'depreciated' should be 'deprecated'.

**Comment by samcv** [comment]
Thanks, typos have been fixed.

**Comment by samcv** [comment]
@jnthn just pushed some new changes. Spectest pass, plus many todo tests as well. Ready for merge.

File: src/strings/unicode_ops.c [hunk 1]
```diff
@@ -107,11 +107,13 @@ MVMint64 MVM_unicode_string_compare
 MVMGrapheme32 MVM_unicode_lookup_by_name(MVMThreadContext *tc, MVMString *name) {
     MVMuint64 size;
     char *cname = MVM_string_ascii_encode(tc, name, &size, 0);
+    size_t cname_len = strlen((const char *) cname );
```
Comment (jnthn) [inline src/strings/unicode_ops.c]: Isn't this already in `size`?

**Pull request merged**